### DISCUSSION TITLE
Make clear in docs that Aes is stateless

### DISCUSSION THREAD

**Discussion by richkzad** [krzyzanowskim/CryptoSwift#924] [General]
It doesn't seem intuitive to me that Aes is a stateless object. You can create an instance of it with a BlockMode, which seems to imply to me that several encrypt/decrypt of the same data would produce different results each time.

If Aes is going to remain stateless, it would be nice to be explicity about this in the docs.

**Comment by krzyzanowskim** [comment]
If you want to create PR with update to README, I'm happy to merge it.

**Comment by richkzad** [comment]
I may eventually... but you may prefer how you write it yourself. I just wanted to let you know it was a bit of a confusion point for me, and it could be for others too. I was happy that this little thing was the only bump and it's working quite splendidly after I figured this out.

An alternative readme-free solution could be to have the iv parameter only in the en/decrypt and makeEn/Decryptor functions, not in init, which would make it more clear that you are starting from the initial block on each call.

**Comment by nathanfallet** [comment]
@richkzad I had the same problem, so I shift my IV by myself and use a new instance of `AES` each time.

See https://github.com/GroupeMINASTE/SwiftMC/blob/d84182be19368f33fb69c1e43ae5a8ea159e70f9/Sources/SwiftMC/Handlers/MinecraftEncoder.swift#L86
